defaults/main.yml

```diff
@@ -9,8 +9,8 @@ install_timeout: "30"
 # Temp directory for storing provisioning related files
 provision_tempdir: "/tmp"
 
-# Where to store libvirt images and VMs
-libvirt_store: "/var/lib/libvirt"
+# Where libvirt stores images and vms
+libvirt_root: "/var/lib/libvirt"
```

Comment:
Looks like this is still set to `/var/lib/libvirt` and not `/var/lib/libvirt/images`. This is fine if you intend to add `/images/` when referring to the image location, but further down the image location is set to `/var/lib/libvirt/{{ fqdn }}.raw`, which breaks `virsh undefine --remove-all-storage` with a default libvirt pool configuration and thus our teardown scripts.

Regardless of how the image path is built, the end result should be that disks are placed under `/var/lib/libvirt/images`, since this is the location of the default storage pool in libvirt. Libvirt will refuse to remove any disks that are outside its own storage pools.